jcl/src/openj9.dtfjview/share/classes/com/ibm/jvm/dtfjview/Session.java

```diff
@@ -614,11 +614,11 @@ private void imageFromCommandLine() {
 			coreFilePath = args.get(ARG_ZIP);	//if -core has been set it is handled in the first part of this if statement
 			ctxroot.execute("open " + coreFilePath , out.getPrintStream());
 		}
-		logger.fine(String.format("Time taken to load image %d ms", System.currentTimeMillis() - current));
+		logger.fine(String.format("Time taken to load image %d ms", System.nanoTime() - current));
```

Comment:
The message indicates "ms", but now it's "ns".

Reply:
Updated.